### PR TITLE
#337 timeline status should be displayed better

### LINKED ISSUE #337: Timeline Status should be displayed better
### Description

Right now we have an enum TimelinStatus (see work-package.types.ts) that displays as like VERY_BEHIND or ON_TRACK or whatever. we should make it more readable like "Very Behind" or "On Track"

### Acceptance Criteria

Make a pipe that converts an enum value to a readable value

Anywhere that these enums are used should be using this new pipe. This includes (but might not be limited to):
* the home page
* work package page

### Proposed Solution

.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by Sybaru**
## Changes

Created a pipe to translate timeline status into a more readable format

## Screenshots

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #337

### CODE REVIEW

File: src/frontend/src/utils/pipes.ts [hunk 1]
```diff
@@ -84,3 +84,17 @@ export const numberParamPipe = (param: string | null) => {
     return null;
   }
 };
+
+/** Display timeline status in readable form
+ *  E.G. VERY_BEHIND -> Very Behind
+ */
+export const timelinePipe = (status: string) => {
+  var res = status.replaceAll('_', ' ');
+  res = res.toLowerCase();
```
Comment (z-roth) [inline src/frontend/src/utils/pipes.ts:93]: looks like a redundant call to `.toLowerCase()` here

File: src/frontend/src/utils/pipes.ts [hunk 2]
```diff
@@ -84,3 +84,17 @@ export const numberParamPipe = (param: string | null) => {
...
+  res = res
+    .toLowerCase()
+    .split(' ')
+    .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(' ');
+  return res;
```
Comment (z-roth) [inline src/frontend/src/utils/pipes.ts:99]: you could instead do: 
`res
.toLowerCase()
.split('_')
.map((s) => s.charAt(0).toUpperCase() +s.substring(1))
.join(' ')`

**Review by z-roth — CHANGES_REQUESTED** [review]

**Review by z-roth — APPROVED** [review]
LGTM - great work!

**Review by RChandler234 — APPROVED** [review]
LGTM!

**Pull request merged**